tests/main/snap-run/task.yaml

```diff
@@ -9,11 +9,9 @@ systems:
   # strace does not support _newselect on s390x
   # (https://github.com/strace/strace/issues/57)
   - -*-s390x
-  # TODO: segfault on core22
-  - -ubuntu-core-22-*
 
 environment:
-    STRACE_STATIC_CHANNEL: beta
+    STRACE_STATIC_CHANNEL: edge
```

Comment:
This passes in tests but is obviously a blocker for actual users.

Reply:
Foundations has agreed to release the snap to stable so I think this can be changed to stable when that moment happens.

Reply:
I've released the snap to candidate. Pushed a patch to trigger CI now.